pkgs/applications/misc/deadd-notification-center/default.nix

```diff
@@ -1,62 +1,61 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, autoPatchelfHook
-, wrapGAppsHook
-, hicolor-icon-theme
-, gtk3
-, gobject-introspection
-, libxml2
-, fetchpatch
-}:
-stdenv.mkDerivation rec {
+{ mkDerivation, haskellPackages, fetchFromGitHub, lib, writeText }:
+
+let
+  systemd-service
```

Comment:
It looks like upstream already provides a `systemd` user service: https://github.com/phuhl/linux_notification_center/blob/master/deadd-notification-center.service.in

You could use that instead.